your-code/main.ipynb

```diff
@@ -167,10 +302,347 @@
+       "4  0.713271  0.156210  0.392800  0.702348  0.009400  0.030304  0.960066   \n",
+       "0  0.734751  0.195362  0.734309  0.598184  0.763433  0.263434  0.868066   \n",
+       "1  0.772607  0.445391  0.249642  0.787922  0.598583  0.827238  0.624126   \n",
+       "2  0.226428  0.268764  0.694262  0.622335  0.063843  0.122683  0.815625   \n",
+       "3  0.362748  0.495430  0.113876  0.594149  0.612522  0.625204  0.864050   \n",
+       "4  0.033415  0.340433  0.464971  0.363737  0.025815  0.434129  0.415163   \n",
+       "\n",
+       "          7         8         9        10        11        12        13  \\\n",
+       "0  0.623167  0.074066  0.111557  0.864664  0.093637  0.446974  0.022525   \n",
+       "1  0.086904  0.912348  0.892183  0.426389  0.776552  0.429496  0.602056   \n",
+       "2  0.921508  0.563333  0.184300  0.667625  0.270017  0.573440  0.163669   \n",
+       "3  0.980700  0.159022  0.684265  0.898633  0.470113  0.756593  0.282408   \n",
+       "4  0.688937  0.260252  0.634608  0.273775  0.264145  0.682941  0.908499   \n",
+       "0  0.058092  0.753502  0.587513  0.311608  0.178356  0.182922  0.147631   \n",
+       "1  0.601524  0.688753  0.338870  0.081595  0.471474  0.267443  0.453351   \n",
+       "2  0.584542  0.032594  0.589775  0.764350  0.650973  0.565705  0.691784   \n",
+       "3  0.260279  0.528873  0.168043  0.715929  0.677014  0.175735  0.632370   \n",
+       "4  0.892210  0.381701  0.415264  0.790801  0.696930  0.819751  0.944029   \n",
+       "\n",
+       "         14        15        16        17        18        19  \n",
+       "0  0.713369  0.707874  0.563506  0.982256  0.562498  0.968681  \n",
+       "1  0.283769  0.166793  0.316878  0.688337  0.169525  0.966198  \n",
+       "2  0.032852  0.689315  0.973684  0.719610  0.163670  0.724486  \n",
+       "3  0.551369  0.732153  0.494100  0.792803  0.794426  0.837695  \n",
+       "4  0.450950  0.200761  0.968324  0.221165  0.891382  0.398914  \n",
+       "0  0.391188  0.816049  0.749068  0.293260  0.937828  0.880858  \n",
+       "1  0.800716  0.045749  0.683793  0.389789  0.016787  0.503695  \n",
+       "2  0.265223  0.739031  0.560394  0.334802  0.517694  0.646110  \n",
+       "3  0.926715  0.085675  0.120525  0.141746  0.771144  0.489660  \n",
+       "4  0.869965  0.041723  0.819140  0.676051  0.109349  0.872947  "
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#os.path.join sirve para unir un path a una lista, y que así sepa referenciar cada archivo al path\n",
+    "csv_list = [file for file in dirlist if file.endswith('.csv')]\n",
+    "print(csv_list)\n",
+    "data_sets = [pd.read_csv(os.path.join(\"/home/eiroamd/Documents/IH/Week1/lab-list-comprehensions/data\", file)) for file in csv_list]\n",
+    "data_frame = pd.concat(data_sets)\n",
+    "data_frame[:10]\n"
```

Comment:
Dani, aquí solamente añadir que puedes hacerlo todo en un paso y que no hace falta que añadas to el path :sweat_smile: 
Si tu path de trabajo es `/home/eiroamd/Documents/IH/Week1/lab-list-comprehensions/your_code` y los datos están en `/home/eiroamd/Documents/IH/Week1/lab-list-comprehensions/data` para acceder a ellos tan sólo te hace falta poner lo siguiente `../data` que viene a ser bajar al directorio padre del que estás actualmente y luego acceder a data. La list comprehension se quedaría tal que así:
```
data_sets = [pd.read_csv(os.path.join("../data", file)) for file in csv_list]
```
O incluso ya hacerlo en una línea:
```
data_frame = pd.concat([pd.read_csv(os.path.join("../data", file)) for file in csv_list],axis=0)
```
Y luego para hacer un print en pantalla de un dataframe y sus primeras 10 rows te diría de usar lo siguiente:
```
data_frame.head(10)
```